app/services/claim_fast_tracking/max_rating_annotator.rb

```diff
@@ -27,6 +29,29 @@ def self.annotate_disabilities(rated_disabilities_response)
       rated_disabilities_response
     end
 
+    def self.log_hyphenated_diagnostic_codes(rated_disabilities)
+      rated_disabilities.each do |dis|
+        Rails.logger.info('Max CFI rated disability',
+                          diagnostic_code: dis&.diagnostic_code,
+                          diagnostic_code_type: diagnostic_code_type(dis&.diagnostic_code),
+                          hyphenated_diagnostic_code: dis&.hyphenated_diagnostic_code)
+      end
+    end
+
+    def self.diagnostic_code_type(diagnostic_code)
```

Comment:
Since you're comparing against the diagnostic_code, a case statement here might improve readability

Reply:
Great suggestion. I had to look up the Ruby syntax for handling the one modulo expression in a case statement, but overall it does look better. Let me know what you think.